images/macos/provision/configuration/preimagedata.sh

```diff
@@ -8,6 +8,9 @@ os_name=$(sw_vers -productName)
 os_version=$(sw_vers -productVersion)
 os_build=$(sw_vers -buildVersion)
 label_version=$(echo $os_version | cut -d. -f1,2)
+if is_BigSur; then
```

Comment:
Two suggestions:
1) include both branches to `if-else` statement to simplify readability.
2) Swap condition branches
```
# preudocode
if is_Less_BigSur {
  label_version=$(echo $os_version | cut -d. -f1,2)
} else {
  label_version=$(echo $os_version | cut -d. -f1)
}
```
So we won't need to edit this condition when macOS 12 / 13 is added

Reply:
Thank you, resolved.